hardhat.config.ts

```diff
@@ -15,7 +15,11 @@ import 'tsconfig-paths/register';
 
 const networks: NetworksUserConfig =
   env.isHardhatCompile() || env.isHardhatClean() || env.isTesting()
-    ? {}
+    ? {
+        hardhat: {
+          chainId: 1, // We are trying to mock mainnet
+        },
+      }
```

Comment:
Do we need this? Kinda weird to set chain id to `1` on local node if its not a fork.

Reply:
Yeah, we need it for our tests. Since we have code that is only activated when the chain is is 1, this is a way to set it. This is what we did on the core repo